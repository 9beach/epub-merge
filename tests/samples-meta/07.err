[90m작가: 배수아 -> BadAuthor; Really ReallyBad Author [But hasLovely Name][0m
[90m도서번호: <empty> -> BAD
ISBN XXXXX[0m
