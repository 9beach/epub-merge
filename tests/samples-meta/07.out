제목: 뱀과 물
작가: Bad
Author

작가: Really Really
Bad Author [But has
Lovely Name]
출판일: 2017-12-07
언어: ko
도서번호: BAD
ISBN XXXXX
